HHS/Head-Start-TTADP
# [TTAHUB-633] Improve display of approval process

## Description of change

A few improvements to the approval process

- Enumerate approvers on the my alerts table
- Changes some button labels ("choose report status" becomes "choose approval status", "re-submit report" becomes "update"
- Fixes to the icons & status badges on the table to make them more accurate

## How to test

- Create a report with two approvers
- The report should say "submitted" in the table and you should see the approvers listed
- As the first approver on the report, approve the report. 
- The "you have reviewed" notification appears above the myalerts table, the icon indicates that you approved, and the badge status says "reviewed"
- As the second approver, mark the report as "needs action" 
- The "you have reviewed" notification appears above the myalerts table, the icon indicates that you marked it as needs action, and the badge status says "needs action"
- Update the report and re-submit it for approval
- Approve the report

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-633


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
